libgstd/gstd_parser.c

```diff
@@ -266,11 +281,13 @@ gstd_parser_create (GstdSession * session, GstdObject * obj, gchar * args,
   if (ret)
     goto out;
 
-  gstd_object_read (obj, name, &new);
+  if (0 != strcmp (name, "eos")) {
+    gstd_object_read (obj, name, &new);
 
-  if (NULL != new) {
-    gstd_object_to_string (new, response);
-    g_object_unref (new);
+    if (NULL != new) {
+      gstd_object_to_string (new, response);
+      g_object_unref (new);
+    }
```

Comment:
why don't you check the object if it is eos?

Reply:
This logic would check if the object was created in GstD after sending EOS and it would always generate an error message after sending EOS because it doesn't generate a new object:
```
pipeline_create p0 videotestsrc ! fakesink
pipeline_play p0
event_eos p0
```


```
0:00:43.282238621 13498   0x7f98003630 ERROR           gstdnoreader gstd_no_reader.c:77:gstd_no_reader_read:<GstdNoReader@0x7fa002a480> Unable to read from this resource
```

Reply:
I think I managed to reproduce this `Unable to read from this resource` error by enabling the python tests in #278.